hadoop-hdds/common/src/main/java/org/apache/hadoop/ozone/audit/SCMAction.java

```diff
@@ -49,7 +49,8 @@ public enum SCMAction implements AuditAction {
   GET_CONTAINER_BALANCER_STATUS,
   GET_CONTAINER_WITH_PIPELINE_BATCH,
   ADD_SCM,
-  GET_REPLICATION_MANAGER_REPORT;
+  GET_REPLICATION_MANAGER_REPORT,
+  RENEW_DELETED_BLOCK_RETRY_COUNT;
```

Comment:
Shall use reset here instead of renew to be consistent?  Actually, I think renew is more precise than reset.

Reply:
Hi Sammi, the latest commit includes this change ~ Now all places use `reset`.